src/app/vm/vm-creation/components/security-group-selector/security-group-selector.component.html

```diff
@@ -19,7 +19,8 @@ <h4 matLine>
         {{ sg.description }}
       </p>
       <span>
-        <mat-radio-button [checked]="sg?.id === securityGroup?.id"></mat-radio-button>
+        <mat-checkbox
+          [checked]="checkSelectedSG(sg?.id)"></mat-checkbox>
```

Comment:
![sg-vm1](https://user-images.githubusercontent.com/2853979/32205833-1861b2da-be24-11e7-9134-448c53e07e70.png)
Sometimes it happens (item is selected but checkbox is not checked)